rpc/serde/jsonrpc.go

```diff
@@ -82,11 +87,16 @@ type NotificationPayload interface {
 		query.LedgerChannelInfo
 }
 
+type Params[T RequestPayload | NotificationPayload] struct {
+	AuthToken string `json:"authtoken"`
```

Comment:
Have you considered the RPC server accepting  the JWT token in using the `Authorization` header ([like infura does](https://docs.infura.io/learn/json-web-token-jwt))? I think that's the "standard" way of including a token in an http request.  Although I guess we do also have to support NATS, so that couldn't be the only way to accept an auth token.

Reply:
HTTP auth headers were the initial implementation. I decided to include the tokens in the RPC params to unify how tokens are sent for all transports.